data/json/npcs/robofac/robofac_intercom/robofac_intercom.json

```
@@ -554,6 +588,233 @@
+      "effect": [
+        { "u_add_var": "u_informed_robofac_of_exodii", "value": "yes" },
+        { "math": [ "hub01_knows_exodii = 1" ] },
+        { "u_add_effect": "robofac_surveillance", "duration": "30 days" },
+        { "u_add_faction_trust": 1 }
+      ]
+    },
+    "responses": [
+      { "text": "Not really.", "topic": "TALK_ROBOFAC_EXODII_MORE_INFO" },
+      {
+        "text": "Not much actually but I do know they have pretty sweet looking technology as I saw them having advanced machinery everywhere. Nothing like I've ever seen.",
+        "topic": "TALK_ROBOFAC_EXODII_MORE_INFO"
+      }
+    ]
+  },
+  {
+    "id": "TALK_ROBOFAC_EXODII_MORE_INFO",
+    "type": "talk_topic",
+    "dynamic_line": "We need more information on these folks, we will be making covert visual contact ourselves but for obvious reasons we can't get inside information without alerting these guys. That's where you come in...we want you to gather as much information on these people without raising any suspicions and bring it back to us. Most importantly find who they are and why are they here.",
+    "responses": [
+      { "text": "Right. I'll be back with the juicy details.", "topic": "TALK_DONE" },
+      {
+        "text": "I don't know man sounds kind of dangerous. What can I expect in return?",
+        "topic": "TALK_ROBOFAC_EXODII_INFO_REWARD"
+      },
+      {
+        "text": "Absolutely not, these guys are far more powerful than me and one wrong move will get me killed, besides I'm no rat.",
+        "topic": "TALK_DONE"
+      }
+    ]
+  },
+  {
+    "id": "TALK_ROBOFAC_ABOUT_EXODII_2",
+    "type": "talk_topic",
+    "dynamic_line": "Inter-dimensional travellers you say? What kind of tech do these beings operate? And how were you able to communicate with them?",
+    "speaker_effect": {
+      "effect": [
+        { "u_add_var": "u_informed_robofac_of_exodii", "value": "yes" },
+        { "math": [ "hub01_knows_exodii = 1" ] },
+        { "u_add_effect": "robofac_surveillance", "duration": "30 days" },
+        { "u_add_faction_trust": 1 }
+      ]
+    },
+    "responses": [
+      { "text": "Alright listen up.", "topic": "TALK_GIVE_ROBOFAC_EXODII_INFO" },
+      { "text": "Information doesn't come cheap my friend.", "topic": "TALK_ROBOFAC_EXODII_INFO_REWARD" },
+      { "text": "I've said more than I ought to have. I'm leaving.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_GIVE_ROBOFAC_EXODII_INFO",
+    "type": "talk_topic",
+    "dynamic_line": "Well?",
+    "speaker_effect": { "effect": [ { "u_add_var": "u_give_robofac_exodii_info", "value": "yes" } ] },
+    "responses": [
+      {
+        "text": "They are a race of inter-dimensional beings who call themseleves the Exodii. I observed they were using bionic enhancements and automated machinery to do their fighting and heavy lifting. They do understand English to a certain extent and when talking they use a strange dialect they like to call Anglic.",
```

```suggestion
        "text": "They are a race of inter-dimensional beings who call themselves the Exodii.  I observed they were using bionic enhancements and automated machinery to do their fighting and heavy lifting.  They do understand English to a certain extent and when talking they use a strange dialect they like to call Anglic.",
```
Double space after period, needs to applied to other strings too. Also it's not obvious they are using bionics in the sense of "mechanical enhancements to a biological body" because most exodii don't have obvious human biology and instead have fully mechanical bodies, so this could be worded differently.

---

Fixed.